src/Core/src/Handlers/Editor/EditorHandler.Android.cs

```diff
@@ -108,6 +108,12 @@ public static void MapPlaceholderColor(IEditorHandler handler, IEditor editor)
 		public static void MapSelectionLength(IEditorHandler handler, ITextInput editor) =>
 			handler.PlatformView?.UpdateSelectionLength(editor);
 
+		static void MapFocus(IEditorHandler handler, IEditor editor, object? args)
+		{
+			if (args is FocusRequest request)
+				handler.PlatformView?.Focus(request);
```

Comment:
SI the focus method the one that was added in the extension? That one already checks for null right? Could we pass it as 
```suggestion
			handler.PlatformView?.Focus(args is FocusRequest);
```